client/src/components/Board/Column.tsx

```diff
@@ -1,16 +1,18 @@
-import { Button, Card, CardActions, CardContent, CardHeader, Modal } from '@material-ui/core';
+import { Button, Card, CardActions, CardContent, CardHeader, Input, Modal } from '@material-ui/core';
 import { Droppable, Draggable } from 'react-beautiful-dnd';
 import useStyles from './useStyles';
 import Task from './Task';
 import { Column as ColumnInterface } from '../../interface/Column';
 import { Task as TaskInterface, TaskPlaceHolder } from '../../interface/Task';
 import { useState } from 'react';
+import DeleteIcon from '@material-ui/icons/Delete';
 export interface properties {
   placeHolderStyle: TaskPlaceHolder;
   Column: ColumnInterface;
```

Comment:
Could this be lowercased to follow camelCase? Change where applicable in the code as well please. 😄

Reply:
I don't think material icons can be camel case.